examples/QKD/e91.py

```diff
@@ -0,0 +1,171 @@
+from qunetsim.backends import EQSNBackend
+from qunetsim.components import Host
+from qunetsim.components import Network
+from qunetsim.objects import Logger
+from qunetsim.objects import Qubit
+
+import random
+import numpy as np
+
+Logger.DISABLED = True
+
+def expected_value(result_string_alice, result_string_bob, bases_string_alice, bases_string_bob, base_alice, base_bob):
+    list = [0, 0, 0, 0]
+
+    for i in range(len(result_string_alice)):
+        if bases_string_alice[i] == base_alice and bases_string_bob[i] == base_bob:
+            if result_string_alice[i] == '0' and result_string_bob[i] == '0':
+                list[0] += 1
+            elif result_string_alice[i] == '0' and result_string_bob[i] == '1':
+                list[1] += 1
+            elif result_string_alice[i] == '1' and result_string_bob[i] == '0':
+                list[2] += 1
+            elif result_string_alice[i] == '1' and result_string_bob[i] == '1':
+                list[3] += 1
+
+    return list
+
+def chsh(result_string_alice, result_string_bob, bases_string_alice, bases_string_bob):
+
+    listA1B1 = expected_value(result_string_alice, result_string_bob, bases_string_alice, bases_string_bob, 'a1', 'b1')
+    listA1B3 = expected_value(result_string_alice, result_string_bob, bases_string_alice, bases_string_bob, 'a1', 'b3')
+    listA3B1 = expected_value(result_string_alice, result_string_bob, bases_string_alice, bases_string_bob, 'a3', 'b1')
+    listA3B3 = expected_value(result_string_alice, result_string_bob, bases_string_alice, bases_string_bob, 'a3', 'b3')
+
+    coefA1B1 = (listA1B1[0] - listA1B1[1] - listA1B1[2] + listA1B1[3]) / sum(listA1B1)
+    coefA1B3 = (listA1B3[0] - listA1B3[1] - listA1B3[2] + listA1B3[3]) / sum(listA1B3)
+    coefA3B1 = (listA3B1[0] - listA3B1[1] - listA3B1[2] + listA3B1[3]) / sum(listA3B1)
+    coefA3B3 = (listA3B3[0] - listA3B3[1] - listA3B3[2] + listA3B3[3]) / sum(listA3B3)
+
+    return coefA1B1 - coefA1B3 + coefA3B1 + coefA3B3
+
+def alice(alice, bob, numberOfEntaglementPairs):
+    angles = [0, np.pi/4, np.pi/2]
+    bases_choice = [random.randint(1,3) for i in range(numberOfEntaglementPairs)]
+    test_results_alice = []
+    test_bases_alice = []
+    sifted_key_alice = []
+
+    for i in range(numberOfEntaglementPairs):
+
+        qubit_a = Qubit(alice)
+        qubit_b = Qubit(alice)
+
+        # preparation of singlet state (1/sqrt(2))*(|01> - |10>)
+        qubit_a.X()
+        qubit_b.X()
+        qubit_a.H()
+        qubit_a.cnot(qubit_b)
+
+        print('Sending EPR pair %d' % (i + 1))
+        _, ack_arrived = alice.send_qubit(bob, qubit_b, await_ack=True)
+        if ack_arrived:
+
+            #rotate qubit and measure
+            base_a = bases_choice[i]
+            qubit_a.rz(angles[base_a-1])
```

Comment:
Operations like `1+1` should have a space: `1 + 1`. Running through an auto-linter should fix this up.